core/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
 
 dependencies {
     compile 'com.novoda:notils:2.2.16'
-    compile 'com.google.android.exoplayer:exoplayer:r1.5.9'
+    compile 'com.google.android.exoplayer:exoplayer:r2.3.1'
```

Comment:
🤔  I assumed we were, they've updated it since the spike and we forgot to check 😟  Will update